### PR TITLE
Check element after scroll

### PR DESCRIPTION
**Pull request by SD1982**

### CODE REVIEW

File: src/utils/pages/commonPage.ts [hunk 1]
```diff
@@ -517,4 +517,29 @@ export class CommonPage {
...
+        if (rect.top >= 0 && rect.left >= 0) {
+          const vw: number = Math.max(document.documentElement.clientWidth || 0, window.innerWidth || 0)
+          const vh: number = Math.max(document.documentElement.clientHeight || 0, window.innerHeight || 0)
+          if (rect.right <= vw && rect.bottom <= vh) {
+            isVisible = true
+          }
+        }
+      }
+      return isVisible
+    }, selector)
```
Comment (boubkerbribri) [inline src/utils/pages/commonPage.ts:543]:
```suggestion
    }, selector);
   
    return isVisible;
```

File: src/utils/pages/commonPage.ts [hunk 2]
```diff
@@ -517,4 +517,29 @@ export class CommonPage {
...
   }
+
+  /**
+   *
+   * Check if an element is visible in viewport after a page scroll
+   * @param page {Page} Browser tab
+   * @param selector {string} Selector to check visibility
+   * Return true if selector visible in viewport and false if not
+   */
+  async isElementVisibleAfterScroll(page: Page, selector: string) {
```
Comment (boubkerbribri) [inline src/utils/pages/commonPage.ts:528]:
```suggestion
  async isElementVisibleAfterScroll(
    page: Page, 
    selector: string
  ): Promise<boolean> {
```

File: src/utils/pages/commonPage.ts [hunk 3]
```diff
@@ -517,4 +517,29 @@ export class CommonPage {
 
     return download.path();
   }
+
+  /**
+   *
+   * Check if an element is visible in viewport after a page scroll
+   * @param page {Page} Browser tab
+   * @param selector {string} Selector to check visibility
+   * Return true if selector visible in viewport and false if not
```
Comment (boubkerbribri) [inline src/utils/pages/commonPage.ts:526]:
```suggestion
   * @return True if selector visible in viewport and false if not
```

File: src/utils/pages/commonPage.ts (hunk 1 above)
Comment (SD1982) [reply src/utils/pages/commonPage.ts:543]: done :+1:

File: src/utils/pages/commonPage.ts (hunk 2 above)
Comment (SD1982) [reply src/utils/pages/commonPage.ts:528]: done :+1:

**Pull request merged**